src/main/scala/mesosphere/marathon/state/AppDefinition.scala

```diff
@@ -517,7 +517,6 @@ object AppDefinition {
     */
   def validNestedAppDefinition(base: PathId): Validator[AppDefinition] = validator[AppDefinition] { app =>
     app.id is PathId.validPathWithBase(base)
-    app.dependencies is every(PathId.validPathWithBase(base))
```

Comment:
`validBasicAppDefinition` doesn't validate the app dependencies. If you remove this line, who validates the paths of the dependencies?